imports/plugins/included/default-theme/client/styles/navbar.less

```diff
@@ -263,6 +263,174 @@ html:not(.rtl) .rui.navbar .languages .dropdown-menu {
   .left(auto);
 }
 
+.rui.navbar .static-pages {
+  .display(flex);
+  .align-items(center);
+  .flex(0 0 auto);
+  height: @navbar-height;
+}
+
+.rui.navbar .static-pages:hover {
+  background-color: @navbar-default-toggle-icon-bar-bg;
+  border-color: @navbar-default-toggle-border-color;
+}
+
+.rui.navbar .static-pages:active {
+  background-color: @navbar-default-toggle-icon-bar-bg;
+  border-color: @navbar-default-toggle-border-color;
+}
+
+.rui.navbar .static-pages .dropdown-toggle {
+  .display(flex);
+  .align-items(center);
+  height: 100%;
+
+  padding: 0 @navbar-padding-vertical;
+  .padding-left(@navbar-padding-vertical);
+}
+
+.rui.navbar .static-pages .dropdown {
+  height: 100%;
+}
+
+.rui.navbar .static-pages .additional-link-container a {
+  color: @link-color;
+}
+
+.rui.navbar .static-pages .additional-link-container a:hover {
+  color: @link-hover-color;
+  background-color: transparent;
+}
+
+.rui.navbar .static-pages .dropdown .circular-icon {
+  width: 40px;
+  height: 40px;
+  margin: -15px 5px;
+  border: 2px solid rgba(129, 122, 122, 0.62);
+  border-radius: 50%;
+  // padding: 10px;
+  display: inline-block;
+}
+
+.rui.navbar .static-pages .dropdown .circular-icon .avatar-initials {
+  line-height: 35px;
+}
+
+.rui.navbar .static-pages .dropdown .user-accounts-dropdown {
+  padding: 10px;
+}
+
+.rui.navbar .static-pages .dropdown .user-accounts-dropdown-content {
+  min-width: 220px;
+  min-height: 50px;
+  // margin-bottom: 30px;
+  color: @gray-darker;
+}
+
+.rui.navbar .static-pages .dropdown ul {
+    padding: 0;
+}
+
+.rui.navbar .static-pages .dropdown .user-accounts-dropdown-apps {
+  margin: 0;
+}
+
+.rui.navbar .static-pages .dropdown .user-accounts-dropdown-apps i {
+  margin: 10px 10px 10px 6px;
+  width: 20px;
+  font-size: inherit;
+  text-align: center;
+ }
+
+.rui.navbar .static-pages .dropdown .dropdown-apps-icon {
+  cursor: pointer;
+  white-space: nowrap;
+  display: block;
+  border-bottom: 1px solid @black10;
+}
+
+.rui.navbar .static-pages .dropdown .dropdown-apps-icon:last-child {
+  border-bottom: none;
+}
+
+.rui.navbar .static-pages .dropdown .dropdown-apps-icon a {
+  display: flex;
+  align-items: center;
+  color: @gray;
+  text-decoration: none;
+  padding: 5px 0;
+  min-height: 44px;
+}
+
+.rui.navbar .static-pages .dropdown .dropdown-apps-icon a:hover,
+.rui.navbar .static-pages .dropdown .dropdown-apps-icon a:focus {
+  color: @brand-vivid-color;
+  text-decoration: none;
+  background-color: transparent;
+}
+
+.rui.navbar .static-pages .dropdown-menu {
+  border: 1px solid @dropdown-border;
+  margin-top: 0;
+  .right(0);
+  .left(auto);
+}
+
+.static-pages-a-tag {
+  display: flex !important;
```

Comment:
Avoid use of `!important`. Make use of css priority (in terms of class naming and position) instead.